plugins/modules/system/parted.py

```diff
@@ -200,9 +200,9 @@
 
 def parse_unit(size_str, unit=''):
     """
-    Parses a string containing a size of information
+    Parses a string containing a size or boundary information
```

Comment:
You also need to update the module's documentation, that negative numbers are supported. Otherwise users won't know except from the changelog fragment.